### PR TITLE
Fix typos in horizontal-pod-autoscale.md

### PR DESCRIPTION
**Pull request by nrakltx**
Fixed typos in HPA docs.

### CODE REVIEW

**Comment by dipesh-rawat** [comment]
@nrakltx Thanks you for your contribution.
Could you please sign the CLA before the PR can be reviewed.
You can follow the steps documented here: https://git.k8s.io/community/CLA.md

**Review by dipesh-rawat — COMMENTED** [review]
/easycla

File: content/en/docs/tasks/run-application/horizontal-pod-autoscale.md [hunk 1]
```diff
@@ -29,8 +29,7 @@ If the load decreases, and the number of Pods is above the configured minimum,
 the HorizontalPodAutoscaler instructs the workload resource (the Deployment, StatefulSet,
 or other similar resource) to scale back down.
 
-Horizontal pod autoscaling does not apply to objects that can't be scaled (for example:
-a {{< glossary_tooltip text="DaemonSet" term_id="daemonset" >}}.)
+Horizontal pod autoscaling does not apply to objects that can't be scaled (e.g. a {{< glossary_tooltip text="DaemonSet" term_id="daemonset" >}}).
```
Comment (kbhawkey) [inline content/en/docs/tasks/run-application/horizontal-pod-autoscale.md:32]: We recommend that you keep the lines wrapped and not use 'e.g.' to replace 'for example'.
Thanks

File: content/en/docs/tasks/run-application/horizontal-pod-autoscale.md [hunk 2]
```diff
@@ -40,7 +39,7 @@ The horizontal pod autoscaling controller, running within the Kubernetes
 desired scale of its target (for example, a Deployment) to match observed metrics such as average
 CPU utilization, average memory utilization, or any other custom metric you specify.
 
-There is [walkthrough example](/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/) of using
+There is a [walkthrough example](/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/) of using
```
Comment (kbhawkey) [inline content/en/docs/tasks/run-application/horizontal-pod-autoscale.md:42]: This is a trivial edit.

File: content/en/docs/tasks/run-application/horizontal-pod-autoscale.md (hunk 1 above)
Comment (windsonsea) [reply content/en/docs/tasks/run-application/horizontal-pod-autoscale.md:32]: Refer to the style guide: https://kubernetes.io/docs/contribute/style/style-guide/#avoid-latin-phrases

**Comment by tengqm** [comment]
/close
closing as a trivial edit.

**Pull request closed without merging**